packages/block-library/src/navigation/deprecated.js

```diff
@@ -52,6 +52,8 @@ const migrateWithLayout = ( attributes ) => {
 };
 
 const v6 = {
+	// `__unstableLocation` was removed in Gutenberg 14.8.
```

Comment:
Is this sufficient for deprecation?

I didn't think an additional deprecation version of the block was warranted, since there is no other form to migrate the block to (besides stripping it of the attribute).